tests/session/session-startTransaction-001.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() mini test
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_server_version('<', '4.0'); ?>
```

Comment:
AFAIK, transactions require a replica set oplog. You should probably add `skip_if_not_replica_set()` here as well.

All session-related stuff requires libmongoc crypto as well, so `skip_if_not_libmongoc_crypto()` should appear as the first "skip if" (for consistency). I believe all of the other session tests have that.

Should this

Reply:
Happy to add the crypto stuff, but why do sessions require it?